assets/src/application/ui/calendars/dateDisplay/BiggieCalendarDate/style.css

```diff
@@ -0,0 +1,185 @@
+/** -------------------------------------------------------------------------
+ * Event Espresso BiggieCalendarDate Stylesheet
+ * © 2020 Event Espresso
+ * ------------------------------------------------------------------------- */
+
+.ee-biggie-calendar-date-bg {
+	box-sizing: border-box;
+	color: var(--ee-color-white);
+	height: 100%;
+	padding: var(--ee-padding-tiny) var(--ee-padding-micro);
+	position: absolute;
+	text-align: center;
+	width: calc(var(--ee-padding-micro) * 34);
+	vertical-align: bottom;
+}
+
+.ee-biggie-calendar-date-bg.ee-bcd-pos-left {
+	float: left;
+	left: 0;
+}
+.ee-biggie-calendar-date-bg.ee-bcd-pos-right {
+	float: right;
+	right: 0;
+}
+
+.ee-biggie-calendar-date-header {
+	font-size: var(--ee-font-size-micro);
+	letter-spacing: var(--ee-letter-spacing-font-size-micro);
+	line-height: var(--ee-font-size-bigger);
+	margin: 0 0 var(--ee-margin-micro);
+}
+
+.ee-biggie-calendar-date .ee-bcd-weekday {
+	font-size: var(--ee-font-size-smaller);
+	font-weight: normal;
+	line-height: var(--ee-font-size-big);
+	letter-spacing: var(--ee-letter-spacing-font-size-tiny);
+}
+
+.ee-biggie-calendar-date .ee-bcd-month {
+	font-size: calc(var(--ee-font-size-tiny) * 2);
+	font-weight: bold;
+	line-height: calc(var(--ee-font-size-tiny) * 2);
+}
+
+.ee-bcd-month-day-sep {
```

Comment:
we might need a helper class at some point

Reply:
for doing what? setting the line height? or adding separators?

Reply:
it uses only `display: none;`
so, maybe just `.hide` class instead